tests/test_core.py

```diff
@@ -189,7 +190,8 @@ class MyObject(object):
 
     assert 'MyObject' == str(value)
     assert '<MyObject>' == repr(value)
-    assert u'обѥѩкт' == unicode(value)
+    if six.PY2:
```

Comment:
Ouh, I'd miss this. I think this is enough to just add `from __future__ import unicode_literals`.